build/tools/scripts/GetSharedFrameworkVersion.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Use env in shebangs for machines where bash is not here.

`#!/usr/bin/env bash`